src/main/steps/common/yourhearings/hearings/content.ts

```diff
@@ -320,6 +322,41 @@ export function getHearingMethod(req: AppRequest, attendees: Attendee[]): string
     hearingMethod = attendees.find(attendee => attendee.partyID === partyData.partyId)!.hearingSubChannel;
   }
   if (hearingMethod !== null) {
+    // const method =[{
+    //   type:"VID",
+    //   value:[HEARING_METHOD.VID,HEARING_METHOD.VIDCVP,HEARING_METHOD.VIDOTHER,HEARING_METHOD.VIDPVL,HEARING_METHOD.VIDSKYPE,HEARING_METHOD.VIDTEAMS,HEARING_METHOD.VIDVHS]
+    // },
+    // {
+    //   type:"TEL",
+    //   value:[HEARING_METHOD.TEL,HEARING_METHOD.TELBTM,HEARING_METHOD.TELCVP,HEARING_METHOD.TELOTHER,HEARING_METHOD.TELSKYP,HEARING_METHOD.TELTEMP]
+    // }]
+    // hearingMethod= method.map(hm => hm.value.find(i=>i==hearingMethod)?hm.type:hearingMethod)
+    // return hearingMethod[0];
+
+    // My suggestion //
```

Comment:
Refactor it and use less complex solutions as discussed.

Reply:
done